src/components/SurveyComponents/FormSelect.tsx

```diff
@@ -0,0 +1,51 @@
+import { FormItem } from '@components'
+import '@styles/SurveyComponents.styles.scss'
+import { useFormContext } from 'react-hook-form'
+
+interface propTypes {
+  name: string
+  label: string
+  isRequired: boolean
+  options: string[]
+  isMultipleChoices: boolean
+}
+
+/**
+ *  FormSelect is a form input component that renders either a Select dropdown or a
+ *  Radio group based on the number of options and allows for single or multiple selections.
+ *  It integrates with react-hook-form for form state and validation management.
+ */
+export function FormSelect({ name, label, options, isRequired, isMultipleChoices }: propTypes) {
```

Comment:
```suggestion
export function FormSelect({ name, label, options, isRequired = false, isMultipleChoices = false }: propTypes) {
```

you can have the isMultipleChoices and isRequired props defaulting to false and have them as optional props